rpcs3/rpcs3qt/syntax_highlighter.cpp

```diff
@@ -52,6 +53,18 @@ void Highlighter::highlightBlock(const QString &text)
 	}
 }
 
+LogHighlighter::LogHighlighter(QTextDocument* parent) : Highlighter(parent)
+{
+	//addRule("^[^·].*$", gui::utils::get_label_color("log_level_always")); // unused for now
```

Comment:
I meant this, you added a regular prefix for "always" logs.

Reply:
but the prefix does not appear in the log files, only in the GUI.
So this regex is not gonna apply here

Reply:
Ah nvm then. My bad.